pctasks/core/pctasks/core/queues.py

```diff
@@ -45,6 +50,8 @@ def from_sas_token(
         account_url: str,
         sas_token: str,
         queue_name: str,
+        message_encode_policy: Optional[MessageEncodePolicy] = None,
```

Comment:
This change from `BinaryBase64EncodePolicy` to `None` means that queues which previously base64 encoded their messages will now not.

I'm not sure where queues are being used at currently.

Note that Azure Functions *requires* that the messages be base64 encoded for queues its monitoring. I'm not sure if we have any of those currently, but we will in the low-latency PR. We'll be able to specify that requirement just that queue though.

Reply:
Grepping through the code, I see `QueueService` used in

- `dev`, which should be fine with these changes
- `router/handlers/{eventgrtid,forward}.py`. I'm unsure, but I *think* this is unused code. I don't see any use of `pctasks.router` outside of `pctasks.router`.
- pctasks/run/workflow/executor/remote.py`. This looks like live code, and might be affected by the change. I will update it to explicitly set the message encode / decode policy.

Reply:
The notification logic in `remote.py` is wired up, but unused by any task. The notification logic was kept around to serve as a basis for PCTasks notifications in the future, but that feature is mostly unimplemented and unused. Changing the message encoding is fine in this case, but feel free to modify the notification logic at will in future changes. The router/handlers etc is also unused, perhaps should be dropped in future PRs as part of a cleanup pass.